### ISSUE TITLE
Fix source icons on the allele tab

### ISSUE THREAD

**Issue by harryhoch** [monarch-initiative/monarch-legacy#165]
The allele tab on the disease page has OMIM  icons that can be hard to read. Furthermore, the OMIM icons link to the Neurolex entry about OMIM as a whole. We should add a text label and link to the OMIM entry for the disease instead.

**Comment by cmungall** [comment]
The consensus seems to be to link directly to the source website for all source links. 

I do think it useful to provide some kind of metadata on _how_ the data came from the source into our systen, what transforms were performed on it, when it was last updated, etc. In theory this should all be stored as metadata we can access, and displayed something like this:

http://beta.neuinfo.org/mynif/databaseList.php?&t=indexable&nif=nlx_151835-1

This page isn't ideal as it says NIF, I think we need a page like this for all NIF/Monarch/SciCrunch

**Comment by mellybelly** [comment]
Note that this is a very important ticket, we absolutely need to make sure there is proper attribution before we can release the Sneak Peak.

We can link out to source the source such as MGI, OMIM, on the jumbotron, as well as the elements on the page that say "source" on the righthand column. It would be nice to have a "annotation info" button or somesuch, that could take advantabe of the metadata as per above? 

For phenotype, MP can go to JAX query. What about HP? simply link to http://www.human-phenotype-ontology.org/?

@sarahjkim ask @nlwashington if questions

**Comment by cmungall** [comment]
For the rows in tables:

Not totally following. Are suggesting two things in the source column for each annotation row. (1) An image of the source database that links to the source database top level page (2) A link called "about" or somesuch that goes to an automatically generated metadata page on our (nif/monarch/SC) system (which will itself of course provide links back to the source database site).

For the jumbotron:

This should be straightforward. I can take care of this

**Comment by harryhoch** [comment]
I think that we should have two links: 1. a link to the source do page for the referenced item, and 2) a link to the derivation data - perhaps "source" and "explanation"?

**Comment by nlwashington** [comment]
sources will now link to the "official" sites (will fall back on neurolex wiki, if available).  still todo - adding additional links to individual data items at external sites (for example, linking to MGI genotype object, when on the genotype page).

**Comment by nlwashington** [comment]
added external link to jumbotron for diseases in 03eee64

**Comment by nlwashington** [comment]
please review and close if satisfied.

**Comment by cmungall** [comment]
Looks good to me!
